misc/vasp/python/processors/Chgcar.py

```diff
@@ -0,0 +1,197 @@
+            if boundary[d] != 0:
+                shift = [0, 0, 0]
+                shift[d] = boundary[d]
+                addVertex(shift)
+        for d1 in range(3):
+            for d2 in range(d1 + 1, 3):
+                if boundary[d1] != 0 and boundary[d2] != 0:
+                    shift = [0, 0, 0]
+                    shift[d1] = boundary[d1]
+                    shift[d2] = boundary[d2]
+                    addVertex(shift)
+        if boundary[0] != 0 and boundary[1] != 0 and boundary[2] != 0:
+            addVertex(boundary)
+
+    mesh = ivw.data.Mesh()
+    mesh.basis = ivw.glm.mat3(basis)
+    mesh.offset = ivw.glm.vec3(offset)
+
+    mesh.addBuffer(ivw.data.BufferType.PositionAttrib, ivw.data.Buffer(
+        numpy.array(position).astype(numpy.float32)))
+    mesh.addBuffer(ivw.data.BufferType.ColorAttrib, ivw.data.Buffer(
+        numpy.array(color).astype(numpy.float32)))
+    mesh.addBuffer(ivw.data.BufferType.RadiiAttrib, ivw.data.Buffer(
+        numpy.array(radius).astype(numpy.float32)))
+    mesh.addBuffer(ivw.data.BufferType.PickingAttrib, ivw.data.Buffer(
+        numpy.array(picking).astype(numpy.uint32)))
+    return mesh
+
+
+class Chgcar(ivw.Processor):
+    def __init__(self, id, name):
+        ivw.Processor.__init__(self, id, name)
+
+        self.volumeOutport = ivw.data.VolumeOutport("chargedensity")
+        self.addOutport(self.volumeOutport, owner=False)
+
+        self.meshOutport = ivw.data.MeshOutport("atoms")
+        self.addOutport(self.meshOutport)
+
+        self.chgcar = ivw.properties.FileProperty("chgcar", "CHGCAR")
+        self.addProperty(self.chgcar)
+
+        self.dataRange = ivw.properties.DoubleMinMaxProperty(
+            "dataRange", "Data Range", 0.0, 1.0, -1.70e308, 1.79e308)
+        self.addProperty(self.dataRange)
+        self.dataRange.semantics = ivw.properties.PropertySemantics("Text")
+        self.dataRange.readOnly = True
+
+        self.useCustomRange = ivw.properties.BoolProperty(
+            "useCustomRange", "Overwrite Data Range", False)
+        self.addProperty(self.useCustomRange)
+
+        self.customDataRange = ivw.properties.DoubleMinMaxProperty(
+            "customDataRange", "Custom Data Range", 0.0, 1.0, -1.70e308, 1.79e308)
+        self.addProperty(self.customDataRange)
+        self.properties.customDataRange.semantics = ivw.properties.PropertySemantics(
+            "Text")
+
+        self.dataset = ivw.properties.StringProperty("dataset", "Data")
+        self.addProperty(self.dataset)
+        self.properties.dataset.readOnly = True
+
+        self.pm = inviwopy.PickingMapper(self, 1, lambda x: self.callback(x))
+
+    @staticmethod
+    def processorInfo():
+        return ivw.ProcessorInfo(
+            classIdentifier="org.inviwo.vasp.Chgcar",
+            displayName="Chgcar Source",
+            category="Source",
+            codeState=ivw.CodeState.Stable,
+            tags=ivw.Tags([ivw.Tag.PY, ivw.Tag("VASP"), ivw.Tag("Volume")])
```

Comment:
How about adding Molecule tag? (or something similar)
Also Mesh since you have Volume.